### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by DPLATA**
This PR will introduce v1 of landing page for card design app. Please make sure everything works correctly if you have any styling suggestions they are welcome.

### CODE REVIEW

File: src/Pages/TemplateSelect.js [hunk 1]
```diff
@@ -1,42 +1,142 @@
 import { Link } from 'react-router-dom';
 import { TemplateCard } from '../components/TemplateCard';
-import styled from 'styled-components';
+// import styled from 'styled-components';
```
Comment (waltergpc) [inline src/Pages/TemplateSelect.js:3]: Let`s avoid having comments of unused stuff

File: src/components/TemplateCard.js [hunk 2]
```diff
@@ -4,26 +4,26 @@ import styled from 'styled-components'
 export const TemplateCard = ({ title }) => {
   return (
     <>
-      <Card>
-        <li>
+      <Wrapper>
+        {/* <li key={key}> */}
```
Comment (waltergpc) [inline src/components/TemplateCard.js:8]: Another comment

**Review by waltergpc — APPROVED** [review]
Little changes requested, either way approved, please just remove those comments

File: src/components/TemplateCard.js (hunk 2 above)
Comment (DPLATA) [reply src/components/TemplateCard.js:8]: will do

File: src/Pages/TemplateSelect.js (hunk 1 above)
Comment (DPLATA) [reply src/Pages/TemplateSelect.js:3]: will do

**Pull request merged**